assets/js/view/company.rb

```diff
@@ -86,14 +87,62 @@ def render
         ]),
       ]
 
-      if @bids
+      if @bids&.length&.positive?
         children << h(:div, { style: bidders_style }, 'Bidders:')
         children << render_bidders
       end
 
       children << h(:div, { style: bidders_style }, "Owner: #{@company.owner.name}") if @company.owner
 
-      h(:div, props, children)
+      h(:div, [
+        h(:div, props, children),
+        render_actions
+      ])
+    end
+
+    def render_actions
+      return h(:div) unless selected? && @game.round.auction?
```

Comment:
don't need this if you add it to the children

also don't use unless with multiple statements, change it to an if